### ISSUE TITLE
broken make install

### ISSUE THREAD

**Issue by karlatec** [nutanix/libvfio-user#529]
When using master branch with commit `0a04aeae6ae904762c3abafb8de9708a9728d503` it is not possible to do `make install`.

```
[libvfio-user]$ make -j16
-- The C compiler identification is GNU 10.2.1
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc - works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Found PkgConfig: /usr/bin/pkg-config (found version "1.6.3")
-- Checking for module 'json-c'
--   Found json-c, version 0.13.1
-- Configuring done
-- Generating done
-- Build files have been written to: /home/klateck/work/spdk/libvfio-user/build/dbg
Scanning dependencies of target pci_caps
Scanning dependencies of target irq
Scanning dependencies of target libvfio-user
Scanning dependencies of target migration
Scanning dependencies of target tran_sock
Scanning dependencies of target pci
Scanning dependencies of target dma
Scanning dependencies of target client
Scanning dependencies of target unit-tests
[  9%] Building C object lib/CMakeFiles/migration.dir/migration.o
[  9%] Building C object lib/CMakeFiles/pci.dir/pci.o
[  9%] Building C object lib/CMakeFiles/pci_caps.dir/pci_caps.o
[ 12%] Building C object lib/CMakeFiles/libvfio-user.dir/libvfio-user.o
[ 16%] Building C object lib/CMakeFiles/irq.dir/irq.o
[ 19%] Building C object lib/CMakeFiles/dma.dir/dma.o
[ 22%] Building C object lib/CMakeFiles/tran_sock.dir/tran_sock.o
[ 25%] Building C object samples/CMakeFiles/client.dir/client.o
[ 29%] Building C object samples/CMakeFiles/client.dir/__/lib/tran_sock.o
[ 32%] Building C object samples/CMakeFiles/client.dir/__/lib/migration.o
[ 35%] Building C object test/CMakeFiles/unit-tests.dir/unit-tests.o
[ 38%] Building C object test/CMakeFiles/unit-tests.dir/mocks.o
[ 48%] Building C object test/CMakeFiles/unit-tests.dir/__/lib/libvfio-user.o
[ 48%] Building C object test/CMakeFiles/unit-tests.dir/__/lib/dma.o
[ 48%] Building C object test/CMakeFiles/unit-tests.dir/__/lib/irq.o
[ 51%] Building C object test/CMakeFiles/unit-tests.dir/__/lib/migration.o
[ 51%] Built target irq
[ 58%] Building C object test/CMakeFiles/unit-tests.dir/__/lib/pci_caps.o
[ 58%] Building C object test/CMakeFiles/unit-tests.dir/__/lib/pci.o
[ 58%] Built target migration
[ 61%] Building C object test/CMakeFiles/unit-tests.dir/__/lib/tran_sock.o
[ 61%] Built target pci
[ 61%] Built target dma
[ 61%] Built target pci_caps
[ 61%] Built target tran_sock
[ 64%] Linking C executable client
[ 64%] Built target client
[ 64%] Built target libvfio-user
[ 67%] Linking C executable unit-tests
Scanning dependencies of target vfio-user-static
Scanning dependencies of target vfio-user-shared
[ 70%] Linking C static library libvfio-user.a
[ 74%] Linking C shared library libvfio-user.so
[ 74%] Built target unit-tests
[ 74%] Built target vfio-user-static
Scanning dependencies of target lspci
Scanning dependencies of target null
Scanning dependencies of target server
[ 74%] Built target vfio-user-shared
[ 77%] Building C object samples/CMakeFiles/lspci.dir/lspci.o
[ 80%] Building C object samples/CMakeFiles/null.dir/null.o
[ 83%] Building C object samples/CMakeFiles/server.dir/server.o
Scanning dependencies of target gpio-pci-idio-16
[ 87%] Building C object samples/CMakeFiles/gpio-pci-idio-16.dir/gpio-pci-idio-16.o
[ 90%] Linking C executable lspci
[ 93%] Linking C executable null
[ 96%] Linking C executable gpio-pci-idio-16
[ 96%] Built target lspci
[ 96%] Built target null
[100%] Linking C executable server
[100%] Built target gpio-pci-idio-16
[100%] Built target server

[libvfio-user]$ sudo make install
make: *** No rule to make target 'install'.  Stop.
```

This also breaks SPDK compilation when building libvfio-user as submodule:
```
[klateck@eagle1 spdk]$ ./configure --with-fio=/usr/src/fio --with-vfio-user --with-rdma --enable-debug
Using default SPDK env in /home/klateck/work/spdk/lib/env_dpdk
Using default DPDK in /home/klateck/work/spdk/dpdk/build
Using 'verbs' RDMA provider
Configuring ISA-L (logfile: /home/klateck/work/spdk/isa-l/spdk-isal.log)...done.
Creating mk/config.mk...done.
Creating mk/cc.flags.mk...done.
Type 'make' to build.
[klateck@eagle1 spdk]$ make -j16
make[2]: *** No rule to make target 'install'.  Stop.
make[1]: *** [Makefile:47: all] Error 2
make: *** [/home/klateck/work/spdk/mk/spdk.subdirs.mk:44: vfiouserbuild] Error 2
make: *** Waiting for unfinished jobs....
```

**Comment by jlevon** [comment]
oops, thanks for letting me know, fixing.

**Comment by jlevon** [comment]
Fixed via #531
